common/event_modifiers/wc_event_modifiers.txt

```diff
@@ -1628,4 +1628,12 @@ barrow_den_average_protection = {
 barrow_den_good_protection = {
 	icon = 5
 	defensive_plot_power_modifier = 0.5
+}
+ulduar_closed_down = {
```

Comment:
As I see, there's no way to remove this modifier, so it makes Loken and Thorim unplayable.

Reply:
it will be removed in event that frees yogg-saron

Reply:
Why this modifier is here right now, but the event will be?

Reply:
cause we cant add ulduar in old god branch

Reply:
Why you added these mehanices that require the OG branch here?